test/schema/artwork/index.js

```diff
@@ -298,6 +298,62 @@ describe("Artwork type", () => {
     })
   })
 
+  describe("#edition_sets", () => {
+    const query = `
+      {
+        artwork(id: "richard-prince-untitled-portrait") {
+          edition_sets {
+            sale_message
+          }
+        }
+      }
+    `
+
+    it("returns the proper sale_message for edition sets", () => {
+      artwork.edition_sets = [
+        {
+          availability: "on hold",
+        },
+        {
+          availability: "on loan",
+        },
+        {
+          availability: "permanent collection",
+        },
+        {
+          price: "$1,000",
+        },
+        {
+          availabilitiy: "not for sale",
```

Comment:
I think it'll be nice to include other fields with each option to have a more realistic view of how things will look in prod. Like when the work is 'On hold' it most likely will still have price, right? This way we can test that we are looking at the fields in the right order.

Reply:
Sure thing.